client/js/stores/rating-store.js

```diff
@@ -0,0 +1,113 @@
+import alt from '../libs/alt';
+import RatingActions from '../actions/rating-actions';
+
+// TODO create tests
+class RatingStore {
```

Comment:
@dwoznicka I don't have idea what is going on here. Why are you storing ```ratings```, ```snippetAvg``` and ```userRatings```?

Generally don't we need just to store a 2 additional informations:
- average rating for snippet
- users ids who rate snippet

And if we have that information inside snippet store it would me probably much easier to manipulate that data (eg. we can count average in the backend)

But I may be missing something :palm_tree: However think if it is the best way we can do it.